nemo/collections/asr/parts/preprocessing/perturb.py

```diff
@@ -785,7 +785,7 @@ def perturb_with_input_noise(self, data, noise, data_rms=None, ref_mic=0, norm_t
         else:
             snr_db = random.uniform(self._min_snr_db, self._max_snr_db)
         if data_rms is None:
-            data_rms = data.rms_db if ref_mic is None else data.rms_db[ref_mic]
+            data_rms = data.rms_db[ref_mic] if isinstance(data.rms_db, (list, np.ndarray)) else data.rms_db
```

Comment:
No, because if you look at the function signature, ref_mic=0 as the default value, and there's also an additional check so that it's never None: https://github.com/NVIDIA/NeMo/blob/2fa4a862e30e6c90a418ed6a8ec3372947f01ccf/nemo/collections/asr/parts/preprocessing/perturb.py#L778